README.markdown

```diff
@@ -131,6 +131,12 @@ An array of services to start, normally `puppet` and `mcollective`. If the array
 
 Alternate source from which you wish to download the latest version of Puppet.
 
+####`install_dir`
+
+The directory the puppet agent should be installed to. This is only applicable for windows operating systems.
+This only applies to an agent upgrade and will not change an already installed agent.
```

Comment:
@mikaelsmith I meant to say that setting the install_dir option will not reinstall an agent package to a new directory. This setting only has effect if the agent on the client is being updated. Does that make any sense?